serving/src/test/java/ai/djl/serving/PythonTranslatorTest.java

```diff
@@ -84,12 +79,22 @@ public void setup() throws ModelException, IOException, ParseException {
                 Files.copy(is, paramFile, StandardCopyOption.REPLACE_EXISTING);
             }
 
-            Path libsDir = modelDir.resolve("bin");
+            Path libsDir = modelDir.resolve("libs");
```

Comment:
Code to include an empty lib for now, for testing. 

Because [ServingTranslatorFactory](https://github.com/deepjavalibrary/djl/blob/master/api/src/main/java/ai/djl/translate/ServingTranslatorFactory.java#L79), if lib/libs folder is not found, then RawTranslator is loaded.